weblab/static/js/main.js

```diff
@@ -269,6 +150,10 @@ function initPage ()
       }
     }
   });
+
+  if ($('#experiment-version').length > 0) {
```

Comment:
This feels like quite a hack - is there a better way provided by `require` for initialising other modules?